handler/transport/backend.go

```diff
@@ -102,7 +102,7 @@ func (b *Backend) initOnce(conf *Config) {
 	b.healthyMu.Unlock()
 
 	// race condition, update possible healthy backend with current origin and hostname
-	b.OnProbeChange(&HealthInfo{Healthy: healthy})
+	b.OnProbeChange(&HealthInfo{Healthy: healthy, State: StateOk.String()})
```

Comment:
I think we should read out the state from healthInfo obj like the healthy bool above. The check could have been run without traffic and may not ok.

Reply:
:joy: I completely mis-read the code… Will change.